arch/x86_64/fentry.S

```diff
@@ -1,8 +1,44 @@
 /* argument passing: %rdi, %rsi, %rdx, %rcx, %r8, %r9 */
 /* return value: %rax */
 /* callee saved: %rbx, %rbp, %rsp, %r12-r15 */
-/* stack frame (with -pg): return addr = 8(%rbp), prev fp = 0(%rbp) */
-/* stack frame (with -fentry): return addr = (%rsp), prev fp = 8(%rsp) */
+/* stack frame (with -pg -mfentry): parent location = addr of 52(%rsp), child ip = 48(%rsp) */
```

Comment:
@namhyung , You mean that we can show stack frame info
at the beginning of `mcount()` or `__fentry__()` like below, right ?

For example:

 - `__fentry__` case
```
(gdb) info f
Stack level 0, frame at 0x7fffffffdd48:
 rip = 0x7ffff7bc9d70 in __fentry__; saved rip = 0x4006b3
 called by frame at 0x7fffffffdd50
 Arglist at 0x7fffffffdd38, args: 
 Locals at 0x7fffffffdd38, Previous frame's sp is 0x7fffffffdd48
 Saved registers:
  rip at 0x7fffffffdd40
```
 - `mcount` case
```
Stack level 0, frame at 0x7fffffffdd30:
 rip = 0x7ffff7bc9ce0 in mcount; saved rip = 0x40069b
 called by frame at 0x7fffffffdd50
 Arglist at 0x7fffffffdd20, args: 
 Locals at 0x7fffffffdd20, Previous frame's sp is 0x7fffffffdd30
 Saved registers:
  rip at 0x7fffffffdd28
```